openslides_backend/action/motion/create.py

```diff
@@ -1,7 +1,50 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.patterns import Collection, FullQualifiedId
 from ..action import register_action
-from ..base import DummyAction
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
 
 
 @register_action("motion.create")
-class MotionCreate(DummyAction):
-    pass
+class MotionCreate(CreateAction):
+    """
+    Create Action for motions.
+    """
+
+    model = Motion()
+    schema = DefaultSchema(Motion()).get_create_schema(
+        properties=[
+            "meeting_id",
+            "title",
+            "number",
+            "state_extension",
+            "sort_parent_id",
+            "category_id",
+            "block_id",
+            "supporter_ids",
+            "tag_ids",
+            "attachment_ids",
+        ],
+        required_properties=["meeting_id", "title"],
+    )
+    schema["items"]["properties"].update({"workflow_id": {"type": ["integer", "null"]}})
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        # calculate state_id from workflow_id
+        workflow_id = instance.pop("workflow_id", None)
+        if workflow_id is None:
+            meeting = self.database.get(
+                FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
+                ["motions_default_workflow_id"],
+            )
+            workflow_id = meeting.get("motions_default_workflow_id")
+        if workflow_id:
```

Comment:
We do need a state id, so in the else-part, you must return an error.

Reply:
Add an else branch which throws an error.
Add a test for this.